homeassistant/components/ollama/conversation.py

```diff
@@ -197,13 +199,20 @@ def _trim_history(self, message_history: MessageHistory, max_messages: int) -> N
                 message_history.messages[0]
             ] + message_history.messages[drop_index:]
 
-    def _generate_prompt(self, raw_prompt: str) -> str:
+    def _generate_prompt(
+        self,
+        raw_prompt: str,
+        device_id: str | None,
+        user_id: str | None,
```

Comment:
Let's remove `user_id` for now, since this is not really the id of the user speaking. We don't have a mechanism yet for identifying speakers.

Reply:
You are right, speech recognition will have no user_id for now.
But what about writing to the assist chat in the app or calling the conversation service some other way with right context?